images/capi/packer/ami/packer.json

```diff
@@ -51,8 +51,6 @@
         "distribution_release": "{{user `distribution_release`}}",
         "distribution_version": "{{user `distribution_version`}}",
         "image_builder_version": "{{user `ib_version`}}",
-        "kubernetes_cni_version": "{{user `kubernetes_cni_semver`}}",
-        "kubernetes_version": "{{user `kubernetes_semver`}}",
```

Comment:
Can we override those values from t1 rather than changing it in our fork?

Reply:
i'll overwrite the tags in t1